src/main/scala/mesosphere/util/state/FrameworkIdUtil.scala

```diff
@@ -18,6 +18,7 @@ class FrameworkIdUtil(mStore: MarathonStore[FrameworkId], timeout: Duration, key
     val frameworkId = FrameworkId(proto.getValue)
     Await.result(mStore.modify(key) { _ => frameworkId }, timeout)
   }
+  def expunge: Future[Boolean] = mStore.expunge(key)
```

Comment:
The convention is to declare and call nullary methods _with_ the parentheses when there are side effects.  See http://docs.scala-lang.org/style/method-invocation.html.